setup.py

```diff
@@ -8,8 +8,11 @@
     'msgpack>=0.5.0',
 ]
 
+needs_pytest = {'pytest', 'test', 'ptr'}.intersection(sys.argv)
+pytest_runner = ['pytest-runner'] if needs_pytest else []
+
 setup_requires = [
-    'pytest-runner'
+    pytest_runner,
 ]
```

Comment:
Should be appended, otherwise you have a nested list there?!

Reply:
Ugh. In my haste :) Thanks!